src/hpx_init_params.cpp

```diff
@@ -11,14 +11,10 @@
 
 
 namespace hpx { namespace detail {
-
-    const hpx::program_options::options_description default_desc(
-                "Usage: " HPX_APPLICATION_STRING " [options]");
     startup_function_type default_startup = startup_function_type();
     shutdown_function_type default_shutdown = shutdown_function_type();
     HPX_MAYBE_UNUSED int dummy_argc = 1;
     // TODO: make it only one parameter, probably add a cast
-    char app_name[] = HPX_APPLICATION_STRING;
     char *default_argv[2] = { app_name, nullptr };
     HPX_MAYBE_UNUSED char **dummy_argv = default_argv;
```

Comment:
Btw, `app_name` is also wrong in `default_argv`. Not many people would probably notice in their applications. And the `default_startup/shutdown` functions aren't needed anymore. So maybe we just move these back into the overloads that need them. The duplication will be mostly removed when the deprecated overloads are gone.

Reply:
On it !